docs/content.zh/docs/ops/upgrading.md

```diff
@@ -518,7 +518,12 @@ Savepoints are compatible across Flink versions as indicated by the table below:
           <td class="text-center"></td>
           <td class="text-center"></td>
           <td class="text-center">O</td>
-          <td class="text-left"></td>
+          <td class="text-left">
+            Due to FLIP-190, Flink 1.15.0/1.15.1 changed the way UIDs are generated for TableAPI 
```

Comment:
Maybe we can use the release notes that I added in the JIRA issue (slightly adapted):
```
For Table API: 1.15.0 and 1.15.1 generated non-deterministic UIDs for operators that make it difficult/impossible to restore state or upgrade to next patch version. A new table.exec.uid.generation config option (with correct default behavior) disables setting a UID for new pipelines from non-compiled plans. Existing pipelines can set table.exec.uid.generation=ALWAYS if the 1.15.0/1 behavior was acceptable due to a stable environment.
```

I tried to be as concise and helpful as possible.

Reply:
Feel free to shorten it if needed for the table.

Reply:
Thanks for the suggestion. I have updated it according to your recommendation, with the addition of a link to the Jira. I do not think it needs shortening, it is not tallest cell in the table. We can always shorten it in the future when it is less relevant.